src/datascience-ui/variable-view/variableViewPanel.tsx

```diff
@@ -22,6 +22,9 @@ export class VariableViewPanel extends React.Component<IVariableViewPanelProps>
 
     constructor(props: IVariableViewPanelProps) {
         super(props);
+
+        // For the variable view we want to start toggled open
```

Comment:
When would the variable view ever want to be closed? Or does closed mean it's not visible? (I'm wondering if it still supports the 'hide/collapse' we have in the other view)

Reply:
When I was implementing this I found the simplest way to do it was to just lift the state and control as directly as possible from its usage in the Interactive Window and old webviews. So it does still have the toggle command available on it. However that toggle command was only triggered by a control on the IW or notebook webview, so it's not accessible for this version of the control.